cogs/starboard.py

```diff
@@ -146,11 +147,10 @@ def get_star_reaction_count(self, message):
         return 0
 
     @commands.command(name='starboardcount', aliases=['starcount'])
-    @commands.has_any_role("aww", "Moderators")
-    async def starboardcount(self, ctx, stars):
+    @commands.has_any_role("Moderators")
```

Comment:
Quite sure the previous restriction was intended; not to be merged until @architdate confirms this.